apps/meteor/tests/e2e/image-gallery.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { createAuxContext } from './fixtures/createAuxContext';
+import { Users } from './fixtures/userStates';
+import { HomeChannel } from './page-objects';
+import { createTargetChannel } from './utils';
+import { expect, test } from './utils/test';
+
+test.use({ storageState: Users.user1.state });
+
+test.describe.parallel('image-gallery', () => {
+	let poHomeChannel: HomeChannel;
+	let targetChannel: string;
+	// Using more than 5 images so that new images need to be loaded by the gallery
+	const imageNames = ['number1.png', 'number2.png', 'number3.png', 'number4.png', 'number5.png', 'number6.png'];
+
+	test.beforeAll(async ({ api, browser }) => {
+		targetChannel = await createTargetChannel(api);
+		const { page } = await createAuxContext(browser, Users.user1);
+		poHomeChannel = new HomeChannel(page);
+
+		await poHomeChannel.sidenav.openChat(targetChannel);
+		await poHomeChannel.content.btnJoinRoom.click();
+		for await (const imageName of imageNames) {
+			await poHomeChannel.content.sendFileMessage(imageName);
+			await poHomeChannel.content.btnModalConfirm.click();
+			await expect(poHomeChannel.content.lastUserMessage).toContainText(imageName);
+		}
+	});
+
+	test.beforeEach(async ({ page }) => {
+		poHomeChannel = new HomeChannel(page);
+
+		await page.goto('/home');
+		await poHomeChannel.sidenav.openChat(targetChannel);
+	});
+
+	test.afterAll(async ({ api }) => {
+		expect((await api.post('/channels.delete', { roomName: targetChannel })).status()).toBe(200);
+	});
+
+	test('expect successfully move to older images by using the left arrow button', async () => {
+		await poHomeChannel.content.lastUserMessage.getByRole('img').click();
+
+		/* eslint-disable no-await-in-loop */
+		for (let i = 0; i < imageNames.length - 1; i++) {
+			await expect(poHomeChannel.content.nextSlideButton).toBeEnabled();
```

Comment:
Can we ensure we are indeed moving to the older images? I mean, the description says it fixes the problem of presenting the images in a reversed way, not sure if this is possible though, but maybe ensuring it by filename? 🤔

Reply:
The way I'd retrieve them to do that would be kinda attached to the order too (since the swiper labels them by their indexes/order), but I think that's better than not validating it at all
Tks for the suggestion, I'll do that

Reply:
Done! Found a way to do that not attached to the order :partying_face: